grades/tests.py

```diff
@@ -1,3 +1,16 @@
-# from django.test import TestCase
+import pytest
+from django.db.models.query import QuerySet
+from grades.models import Grade
 
-# Create your tests here.
+
+@pytest.mark.django_db
+class TestGrade:
+    def test_addGradeUserCourse(cls):
+        out = Grade.objects.all()
+        assert isinstance(out, QuerySet)
+        assert all(isinstance(g, Grade) for g in out)
```

Comment:
The previous 2 asserts are testing the base django functionality. No real need to do this.